docs/sample_config.yaml

```diff
@@ -1973,3 +1973,20 @@ opentracing:
     #
     #  logging:
     #    false
+
+
+## Cache Configuration ##
+
+# Caching can be configured through the following options.
+#
+caches:
+   # Controls the global cache factor. This overrides the "SYNAPSE_CACHE_FACTOR"
+   # environment variable.
```

Comment:
also: is it still true that this overrides the env?

Reply:
Ah, no it's not!